src/Controller/AuthenticationController.php

```diff
@@ -42,7 +42,7 @@ public function redirectAtLogin(Request $request, array $requestAttributes): Res
         $password = $request->getParameter('password');
         $role = $this->authenticationService->login($email, $password);
         $userRepo = $this->repositoryManager->getRepository(User::class);
-        $user = $userRepo->findOneBy(['email' => $email, 'password' => $password]);
+        $user = $userRepo->findOneBy(['email' => $email, 'password' => md5($password)]);
```

Comment:
you should not place `md5` here. what if you want to change the hashing strategy? better make a `HashingService` (or how you want to call it) that deals with the hashing. 
Think about Strategy pattern and how at some point you would like to change the strategy in a future version